### ISSUE TITLE
Update bot or change PDF conversion pipeline

### ISSUE THREAD

**Issue by ericcornelissen** [simple-icons/svg-to-pdf-bot#3]
As was discovered by multiple people, e.g. https://github.com/simple-icons/svg-to-pdf-bot/issues/2 and https://github.com/simple-icons/simple-icons/pull/4966#issuecomment-781328267, this bot is no longer working properly. I think we have two options here:

- Fix this bot so that it can be used again
- Change how our [SVGs](https://github.com/simple-icons/simple-icons) are converted into [PDFs](https://github.com/simple-icons/simple-icons-pdf), e.g. through a GitHub Actions workflow setup similar to [the font](https://github.com/simple-icons/simple-icons-font)

I would personally be in favor of the latter, as I don't really want to fix this bot up :sweat_smile: But if someone else is interested in that, go ahead :smile:

**Comment by PeterShaggyNoble** [comment]
I agree with moving to GH Actions if we can so that we're consistently using it across all our repos.
